### PR TITLE
README.md: update and copy tweaks

### PR DESCRIPTION
**Pull request by marcelomazza**
- There was a mix of tabs and spaces; changed everything to tabs
- Changed format of markdown's headings for consistency
- Replaced all the hardcoded ProjectId with `<YOUR PROJECT ID>`
- Replaced all the hardcoded ProjectKey with `<YOUR API KEY>`
- Added Environment to default examples
- Added section headers
- Sorted options by importance
- Added Negroni example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,8 +72,9 @@ import (
 )
 
 var airbrake = gobrake.NewNotifierWithOptions(&gobrake.NotifierOptions{
-	ProjectId:  105138,
-	ProjectKey: "fd04e13d806a90f96614ad8e529b2822",
+	ProjectId:  <YOUR PROJECT ID>,
```
Comment (shifi) [inline README.md:75]:
```suggestion
	ProjectId: <YOUR PROJECT ID>,
```

File: README.md [hunk 2]
```diff
@@ -88,19 +84,24 @@ func main() {
 }
 ```
 
-Configuration
--------------
+To find `<YOUR PROJECT ID>` and `<YOUR API KEY>` navigate to your project's
+Settings and copy the values from the right sidebar.
+
+
+
```
Comment (shifi) [inline README.md:91]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -112,35 +113,26 @@ recommended way to configure your notifier.
 
 ```go
 airbrake := gobrake.NewNotifierWithOptions(&gobrake.NotifierOptions{
-	ProjectId:  105138,
-	ProjectKey: "fd04e13d806a90f96614ad8e529b2822",
+	ProjectId:  <YOUR PROJECT ID>,
```
Comment (shifi) [inline README.md:116]:
```suggestion
	ProjectId: <YOUR PROJECT ID>,
```

File: README.md [hunk 4]
```diff
@@ -261,42 +264,49 @@ notice.Context["severity"] = "critical"
...
-  "github.com/airbrake/gobrake"
+	"github.com/airbrake/gobrake"
 )
 
 // Airbrake is used to report errors and track performance
 var Airbrake = gobrake.NewNotifierWithOptions(&gobrake.NotifierOptions{
-  ProjectId:   123123,				// <-- Fill in this value
-  ProjectKey:  "YourProjectAPIKey", // <-- Fill in this value
-  Environment: "Production",
+	ProjectId:   <YOUR PROJECT ID>,				// <-- Fill in this value
```
Comment (shifi) [inline README.md:292]: Do we need to align it like this? Maybe we could just do this:
```suggestion
	ProjectId: <YOUR PROJECT ID>, // <-- Fill in this value
```

Comment (shifi) [reply README.md:292]: Same thing for other similar lines here.

File: README.md [hunk 5]
```diff
@@ -266,7 +266,7 @@ airbrake.Notify(notice, nil)
 
 ### Performance Monitoring
 
-You can read more about our [Performance Monitoring offering in our docs](docs/performance).
+You can read more about our [Performance Monitoring offering in our docs][docs/performance].
```
Comment (shifi) [inline README.md:269]: This link 404's for me. Does it work for you?

Comment (marcelomazza) [reply README.md:269]: Yes, it works when I preview the file in GitHub. Are you testing it on GitHub?

Comment (shifi) [reply README.md:266]: It works now for me as well. Sry, I'm not sure what happened before :)

**Pull request merged**